usaspending_api/awards/models.py

```diff
@@ -828,3 +828,96 @@ class Subaward(DataSourceTrackedModel):
     class Meta:
         managed = True
         unique_together = (('subaward_number', 'award'),)
+
+
+class SummaryAwardView(models.Model):
+
+    # Fields
+    action_date = models.DateField(blank=True, null=True)
+    fiscal_year = models.IntegerField()
+    type = models.TextField(blank=True, null=True)
+    category = models.TextField(blank=True, null=True)
+    awarding_agency_id = models.TextField(blank=True, null=True)
+    awarding_agency_name = models.TextField(blank=True, null=True)
+    awarding_agency_abbr = models.TextField(blank=True, null=True)
+    funding_agency_id = models.TextField(blank=True, null=True)
+    funding_agency_name = models.TextField(blank=True, null=True)
+    funding_agency_abbr = models.TextField(blank=True, null=True)
+    federal_action_obligation = models.DecimalField(max_digits=20, db_index=True, decimal_places=2, blank=True,
```

Comment:
Does indexing the federal_action_obligation improve performance? Seems odd to me